config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```
@@ -265,6 +265,261 @@ presubmits:
       testgrid-alert-email: antonio.ojea.garcia@gmail.com, danwinship@redhat.com
       description: Uses kubetest to run e2e Conformance, SIG-Network tests against a cluster using nftables created with cluster/kube-up.sh
 
+  - name: pull-kubernetes-e2e-gce-kubedns-performance
+    cluster: k8s-infra-prow-build
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    always_run: false
+    run_if_changed: '^(test/e2e/network/|cluster/addons/dns/kube-dns/)'
+    optional: true
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    decorate: true
+    decoration_config:
+      timeout: 80m
+    spec:
+      containers:
+      - command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --cluster=gce-kubedns-perf
+        - --env=CLUSTER_DNS_CORE_DNS=false
+        - --env=KUBE_ENABLE_NODE_PROBLEM_DETECTOR=none
+        - --extract=ci/latest
+        - --gcp-master-image=gci
+        - --gcp-node-image=gci
+        - --gcp-nodes=3
+        - --gcp-zone=us-east1-b
+        - --provider=gce
+        - --test_args=--ginkgo.focus=\[Feature:PerformanceDNS\]
+        - --timeout=60m
+        - --use-logexporter
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240515-17c6d50e24-master
+        resources:
+          limits:
+            cpu: 2
+            memory: "6Gi"
+          requests:
+            cpu: 2
+            memory: "6Gi"
+    annotations:
+      testgrid-dashboards: sig-network-gce
+      testgrid-tab-name: gce-kubedns-performance
+
+  - name: pull-kubernetes-e2e-gce-kubedns-performance-nodecache
+    cluster: k8s-infra-prow-build
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    always_run: false
+    run_if_changed: '^(test/e2e/network/|cluster/addons/dns/kube-dns/)'
```

Do you mean 'nodelocaldns' addon? Or both `kube-dns` and `nodelocaldns`?
```suggestion
    run_if_changed: '^(test/e2e/network/|cluster/addons/dns/nodelocaldns/)'
```

---

Well-spotted, I intended to write `cluster/addons/dns/` to cover both sub-directories, but apparently made a typo. In any case, Antonio [asks](https://github.com/kubernetes/test-infra/pull/33962/files#r1886855534) to get rid of `run_if_changed` entirely.